packages/svelte-ux/src/routes/+layout.svelte

```diff
@@ -32,61 +33,32 @@
   $: title = data.pr_id ? `🚧 (pr:${data.pr_id}) - ${baseTitle}` : baseTitle;
 
   settings({
-    // Usefull to test different locales with the docs
-    // formats: {
-    //   numbers: {
-    //     defaults: {
-    //       locales: 'fr',
-    //       currency: 'EUR',
-    //     },
-    //   },
-    //   dates: {
-    //     locales: 'fr',
-    //     weekStartsOn: 1,
-    //     presets: {
-    //       days: {
-    //         long: { dateStyle: 'full' },
-    //       },
-    //       months: {
-    //         default: [DateToken.Month_long],
-    //       },
-    //     },
-    //     ordinalSuffixes: {
-    //       fr: {
-    //         one: 'er',
-    //         two: '',
-    //         few: '',
-    //         other: '',
-    //       },
-    //     },
-    //   },
-    //   dates: {
-    //     locales: 'fr',
-    //     weekStartsOn: 1,
-    //     presets: {
-    //       days: {
-    //         long: { dateStyle: 'full' },
-    //       },
-    //       months: {
-    //         default: [DateToken.Month_long],
-    //       },
-    //     },
-    //     ordinalSuffixes: {
-    //       fr: {
-    //         one: 'er',
-    //         two: '',
-    //         few: '',
-    //         other: '',
-    //       },
-    //     },
-    //   },
-    // },
-    // dictionary: {
-    //   Cancel: 'Annuler',
-    //   Date: {
-    //     Day: 'Jour',
-    //   },
-    // },
+    // fallbackLocale: 'fr',
+    localeFormats: {
+      fr: createLocaleSettings({
+        locale: 'fr',
+        formats: {
+          dates: {
+            baseParsing: 'dd/MM/yyyy',
+            ordinalSuffixes: {
+              one: 'er',
+            },
+          },
+          numbers: {
+            defaults: {
+              currency: 'EUR',
+            },
+          },
+        },
```

Comment:
So now `locale` is for dates & numbers at the same time? For my use case it's fine, just want to be sure it's intentional.

At the same time, we repeat 2 times "fr". Maybe `localesFormats` should be an array?

Reply:
If you're fine with the single locale, I definitely am.  I'm not aware of a use case of where we would need to have them separate, especially from a UI/selection standpoint.  There would have to be a really strong reason to support individual locales per date/number/etc :) 

The object makes it easier/faster to lookup the value instead of `arr.find(...)`.

Reply:
Switzerland is a good example of complicated locales ^^ with some french, some german, currency in CHF... 😅

But yes nvm for now, and it can evolve, so all good.